src/main/plugins/org.dita.html5/xsl/topic.xsl

```diff
@@ -548,7 +548,7 @@ See the accompanying LICENSE file for applicable license.
     <xsl:choose>
       <xsl:when test="@othertype">
         <xsl:apply-templates select="." mode="process.note.common-processing">
-          <xsl:with-param name="type" select="'note'"/>
+          <xsl:with-param name="type" select="replace(normalize-space(@othertype), ' ', '_')"/>
```

Comment:
This assumes that the othertype is a token, rather than a string; many of the cases I've seen where `@othertype` is used are full natural-language titles. So we would end up with something like a type parameter of `My_type_of_note`. That othertype value is already passed in the next `title` parameter.

In the common processing, this will assume that the variable `My_type_of_note` is defined, and throw an error if it is not. Today we print out the string (and I know some tools translate it appropriately), while this change would result in broken output until the variable is defined.